pages/common/autojump.md

```diff
@@ -2,6 +2,7 @@
 
 > Quickly jump among the directories you visit the most.
 > Aliases like j or jc are provided for even less typing.
+> TAB key can be used to list the top matches.
```

Comment:
Our guidelines only admit up to two lines in the main description. I'd rather we don't open a precedent against that here. Could we maybe condense the second and third lines into a single one?